wireapp/kalium
# test: add support for test video capture and rendering to Kalium CLI (WBP-2101)

* Adds support for video capture and rendering with real capturer-renderers (OSX) and fake capturer-renderers (OSX & Linux) in Kalium-cli.
* Updates AVS dependency to 9.7.1 to make use of new kcall API providing above.

#### How to Test

Run Kalium CLI with "console" option, make call, use V command to toggle outgoing video. Incoming video is shown for real renderers automatically.
Add "-T" option to enable testing mode (auto-generated video capture and dummy renderers).
